Gems/PhysX/Code/Source/PrismaticJointComponent.cpp

```diff
@@ -84,4 +87,83 @@ namespace PhysX
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
     }
+
+    // AZ::Component
+    void PrismaticJointComponent::Activate() {
+        JointComponent::Activate();
+        PhysX::JointInterfaceRequestBus::Handler::BusConnect(this->GetEntityId());
+    };
+
+    void PrismaticJointComponent::Deactivate() {
+        JointComponent::Deactivate();
+        PhysX::JointInterfaceRequestBus::Handler::BusDisconnect();
+    };
+
+    physx::PxD6Joint* GetPhysXD6Joint(const AzPhysics::JointHandle& joint_handle,
+                                                        const AzPhysics::SceneHandle& joint_scene_owner)
+    {
+        auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        const auto joint = sceneInterface->GetJointFromHandle(joint_handle,joint_scene_owner);
+        physx::PxJoint* native = static_cast<physx::PxJoint*>(joint->GetNativePointer());
+        if (native && native->is<physx::PxD6Joint>())
+        {
+            physx::PxD6Joint *native_d6 = static_cast<physx::PxD6Joint*>(joint->GetNativePointer());
+            return native_d6;
+        }
+        return nullptr;
+    }
+
+    float PrismaticJointComponent::GetPosition(){
+        auto* native = GetPhysXD6Joint(m_jointSceneOwner,m_jointHandle);
+        if (native){
+            return native->getRelativeTransform().p.x;
+        }
+        return 0.f;
+    };
+
+    float PrismaticJointComponent::GetVelocity() {
+        auto* native = GetPhysXD6Joint(m_jointSceneOwner,m_jointHandle);
+        if (native){
+            return native->getRelativeLinearVelocity().x;
+        }
+        return 0.f;
+    };
+
+
+    AZStd::pair<float, float> PrismaticJointComponent::GetLimits(){
+        auto* native = GetPhysXD6Joint(m_jointSceneOwner,m_jointHandle);
+        if (native){
+            auto limits = native->getLinearLimit(physx::PxD6Axis::eX);
+            return AZStd::pair<float, float>(limits.lower,limits.upper);
+        }
+        return AZStd::pair<float, float> (0.f,0.f);
```

Comment:
uber nit: Could use `AZStd::make_pair` here and avoid the need for template args

(this actually should work without as-is due to CTAD in C++17 I believe, but I could be wrong 🤔)